packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -142,6 +143,113 @@ class DateUtil {
     return undefined;
   }
 
+  /**
+   * Returns an easily parsable date format string.
+   * @param {string} locale - The locale variant from dateFormatOrder.
+   */
+  static getDateFormat(locale) {
+    let dateVariant;
+
+    if (locale === '0') {
```

Comment:
When will locale ever be 0? Shouldn't this be a string value representing the translation and locale data loaded by terra-i18n?

Reply:
This is not the locale. This is Terra.datePicker.dateFormatOrder from the translation files. I think we should give it a better name.

Reply:
I can rename this to something easier to reason about. :)